eng/pipelines/runtime.yml

```diff
@@ -185,7 +185,7 @@ jobs:
     buildConfig: debug
     platforms:
     - Linux_x64
-    - windows_x64
+    - Linux_arm64
```

Comment:
Is this intentional change?

Reply:
For CI builds, we want to maximize variety in the subset of configurations that are being tested. This is reducing the variety.

Reply:
Yes, the Linux arm64 container was timing out and I wanted to check with windows. @MichalStrehovsky has mentioned an issue impacting running multiple library test projects (I think but not sure) that should be fixed in 7.0 and I wanted to see if this will work

Reply:
Is this a temporary change that you are going to revert before merging the PR, or is your intention to merge this change?

Also, notice that the runtime jobs did not run at all and no NativeAOT testing was done in the latest CI run. Not sure what caused it.

Reply:
This was to be a diagnostic effort and then to decide if this needs to be permanent. Yes, this is blocked by the lack of full test pass and not sure what is causing that.

Reply:
CoreCLR tests don't work on arm64. We're still running those tests on the build machine and the build machine is x64.

Only libs tests can currently run on arm64 because we send those to helix to run.

Reply:
The most recent test run has completed successfully but I'll revert to the previous position to double check - the earlier runs kept on failing on arm64 for lib tests.

Reply:
All the checks pass with the original runtime pipeline now